backend/run.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask, render_template, jsonify
+from random import *
+import requests
+
+app = Flask(__name__,
+            static_folder = "../frontend/dist/static",
+            template_folder = "../frontend/dist")
+
+@app.route('/api/v1/nodes')
+def random_number():
+    response = {
+        'nodes': ['node0', 'node1', 'node2']
+    }
+    return jsonify(response)
+
+@app.route('/', defaults={'path': ''})
+@app.route('/<path:path>')
+def catch_all(path):
```

Comment:
Should we also consider configuring MyPy for static type checking? A full setup example is also available in the PyDSDL repo: https://github.com/UAVCAN/pydsdl